packages/design-tokens/docs/pages/how-to-use-in-react.stories.mdx

```diff
@@ -11,20 +9,29 @@ import gettingStartedPage from './getting-started.stories.mdx'
 # How to use design tokens in React components
 
 
-## 1. Setup a ThemeManager
+## 1. Setup a KaizenProvider
 
-Ensure you have followed the steps in <LinkToStory storyModule={gettingStartedPage} /> to setup your application for using design tokens.
+Ensure you have followed the steps in [KaizenProvider](/story/components-kaizenprovider--page) to setup your application for using themes.
 
-## 2. Wrap your application in a `ThemeProvider`
+## 2. Start using design tokens with the `useTheme` hook
 
-Import `ThemeProvider` from design tokens, and wrap your entire application with it, providing it the ThemeManager instance you created in the previous steps.
+Once the theme context is available, the `useTheme` hook will allow you to subscribe components and give access to the current theme, and re-render whenever it changes.
 
-Example: 
-<CodeBlock language="typescript" code={indexTsx} caption="index.tsx" />
+Example:
 
-## 3. Start using design tokens with the `useTheme` hook
+```jsx
+import { useTheme } from "@kaizen/design-tokens"
 
-Once the theme context is available, the `useTheme` hook will allow you to subscribe components and give access to the current theme, and re-render whenever it changes.
+const Component = () => {
+  const theme = useTheme();
+
+  return (
+    <div style={{
+      backgroundColor: theme.color.purple["500"] // #844587
+    }}>
+      Theme'd component
```

Comment:
```suggestion
      Themed component
```
I just can't get past it :') Any way to restructure so we use `theme` and not some variation?